fab/operations/release.py

```diff
@@ -143,10 +144,14 @@ def offline_pip_install():
         env.sudo_user, env.virtualenv_root
     )
     requirements = os.path.join(env.code_root, 'requirements')
+    requirements_file = os.path.join(requirements, 'requirements.txt')
+    for offline_lib in ['pygooglechart', 'django-transfer', 'django-two-factor-auth', 'pyzxcvbn']:
```

Comment:
Is this just because the requirements of these are listed as either git or https? If so, we might be able to split those out to a separate reqs file